chaco/__init__.py

```diff
@@ -3,6 +3,9 @@
 """ Two-dimensional plotting application toolkit.
     Part of the Chaco project of the Enthought Tool Suite.
 """
-from ._version import full_version as __version__  # noqa
+try:
+    from chaco._version import full_version as __version__  # noqa
+except ImportError:
+    __version__ = "not-built"
 
-__requires__ = ["traits", "traitsui", "pyface", "numpy", "enable"]
+__requires__ = ["traits", "traitsui", "pyface", "numpy", "enable @ git+http://github.com/enthought/enable.git#egg=enable"]  # noqa
```

Comment:
```suggestion
__requires__ = ["traits", "traitsui", "pyface", "numpy", "enable"]
```

we can just make this change and we should be good to go with this PR